src/makeInfinityAdder.js

```diff
@@ -5,6 +5,23 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const adder = (b) => {
+    if (b === undefined) {
+      const finalSum = sum;
+
+      sum = 0;
+
+      return finalSum;
+    }
+
+    sum += b;
```

Comment:
It's good that you're directly adding the value to 'sum'. However, it's important to consider that if 'b' is not a number, this could lead to unexpected results. Even though the task specifies not to check for the argument type, in real-world scenarios, you might want to add a type check here.